client/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable import/no-named-as-default-member */
+/* eslint-disable import/extensions */
+import React from 'react';
+import ReactDom from 'react-dom';
+import SearchBar from './components/SearchBar.jsx';
+
+ReactDom.render(<SearchBar />, document.getElementById('app'));
```

Comment:
I think we need to be careful about these ID's since we will all be appending to the same HTML document.

Reply:
good point - I'll change it to SearchBar